src/QsCompiler/Tests.LanguageServer/Tests.LanguageServer.csproj

```diff
@@ -27,6 +27,9 @@
     <ProjectReference Include="..\LanguageServer\LanguageServer.csproj">
       <Private>true</Private>
     </ProjectReference>
+    <ProjectReference Include="..\Compiler\Compiler.csproj">
+      <Private>true</Private>
+    </ProjectReference>
```

Comment:
Using CompilationLoader for `TestUtils.BuildDll()` means we need to have the language server tests have a reference to the compiler project. I'm not sure how big of a deal this is

Reply:
I don't see an issue with it.